java/ql/src/semmle/code/java/dataflow/internal/DataFlowImpl.qll

```diff
@@ -731,33 +738,41 @@ private predicate flowIntoCallNodeCand1(
 }
 
 private module Stage2 {
+  class ApApprox = Stage1::Ap;
```

Comment:
It ends up as `class ApApprox = PrevStage::Ap;` in a later commit (I tried minimising commits with changes that were obsoleted by later commits, but it was hard to avoid completely).

Reply:
Right, but what I meant was I am not a fan of the `ApApprox` name. I would rather have `PrevStageAp` or just `PrevStage::Ap`.